platform/landing-page/src/main/java/org/codice/ddf/landing/LandingPage.java

```diff
@@ -269,8 +245,10 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
     }
 
     // package-private for unit testing
-    String compileTemplateWithProperties() {
-        logoToUse = StringUtils.isNotEmpty(logo) ? logo : productImage;
+    String compileTemplateWithProperties() throws IOException {
+        title = getTitle();
+        version = branding.getAttributeFromBranding(BrandingPlugin::getProductName);
```

Comment:
No, the `getVersion` returns the number (eg "v1.0.0") and the `getName` returns the name, eg "DDF" but the `BrandingPlugin::getProductName` returns the "whole thing" eg `DDF v1.0.0` which I think was the intent on this page.